serial_port: /dev/ttyUSB0
baudrate: 15000

head_yaw_driver:
    joint_name: head_yaw_joint
    id: 19
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

left_shoulder_pitch_driver:  
    joint_name: left_shoulder_pitch_joint
    id: 16
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

left_shoulder_roll_driver:   
    joint_name: left_shoulder_roll_joint
    id: 17
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.25

left_elbow_pitch_driver:     
    joint_name: left_elbow_pitch_joint
    id: 18
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

left_waist_yaw_driver:       
    joint_name: left_waist_yaw_joint
    id: 15
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

left_waist_roll_driver:      
    joint_name: left_waist_roll_joint
    id: 14
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

left_waist_pitch_driver:     
    joint_name: left_waist_pitch_joint
    id: 13
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

left_knee_pitch_driver:      
    joint_name: left_knee_pitch_joint
    id: 12
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

left_ankle_pitch_driver:     
    joint_name: left_ankle_pitch_joint
    id: 11
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

left_ankle_roll_driver:      
    joint_name: left_ankle_roll_joint
    id: 10
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

right_shoulder_pitch_driver:  
    joint_name: right_shoulder_pitch_joint
    id: 7
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: -0.28

right_shoulder_roll_driver:   
    joint_name: right_shoulder_roll_joint
    id: 8
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: -0.25

right_elbow_pitch_driver:     
    joint_name: right_elbow_pitch_joint
    id: 9
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

right_waist_yaw_driver:       
    joint_name: right_waist_yaw_joint
    id: 6
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

right_waist_roll_driver:      
    joint_name: right_waist_roll_joint
    id: 5
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

right_waist_pitch_driver:     
    joint_name: right_waist_pitch_joint
    id: 4
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

right_knee_pitch_driver:      
    joint_name: right_knee_pitch_joint
    id: 3
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.0

right_ankle_pitch_driver:     
    joint_name: right_ankle_pitch_joint
    id: 2
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: -1
    offset: 0.25

right_ankle_roll_driver:      
    joint_name: right_ankle_roll_joint
    id: 1
    min_angle: -180
    max_angle: 180
    stretch: 60
    speed: 127
    current_limit: 3000
    temperature_limit: 80
    direction: 1
    offset: 0.0

